app/javascript/controllers/campaign_controller.js

```diff
@@ -2,7 +2,7 @@ import { Controller } from "stimulus"
 import consumer from "../channels/consumer"
 
 export default class extends Controller {
-  static targets = ["mapOption", "currentMap"]
+  static targets = ["mapOption", "currentMap", "image"]
```

Comment:
From an earlier experiment?

Reply:
Yes, good catch, thanks. Removed in 44f6aa1.